### PR TITLE
update tautulli to use dropsignal repo instead of arcooke repo

### PR DESCRIPTION
**Pull request by dropsignal**
Thank you for your submission to the FreeNAS community plugins repository!

Please ensure the following guidelines are met when submitting a new Plugin.

1. Add the entry to INDEX in alphabetical order
2. Includes a new icon in the ./icon/ directory
3. Ensure the submitted icon file is a valid PNG that is 128x128 in size
4. Add new plugin cirrus task

### CODE REVIEW

**Comment by dropsignal** [comment]
I've reviewed the error and I forgot to change the permissions for post-install.sh to 0755. I've corrected that in my fork, but I'm uncertain how to proceed with updating this pull request to grab the new commit.

File: tautulli.json [hunk 1]
```diff
@@ -1,12 +1,8 @@
 {
     "name": "Tautulli",
     "release": "12.2-RELEASE",
-    "artifact": "https://github.com/arcooke/iocage-plugin-tautulli.git",
+    "artifact": "https://github.com/dropsignal/iocage-plugin-tautulli.git",
     "official": false,
-    "properties": {
-        "nat": 1,
```
Comment (sonicaj) [inline tautulli.json:7]: This plugin should not default to using NAT anymore ?

**Review by sonicaj — CHANGES_REQUESTED** [review]
`post-install.sh` should be `post_install.sh` and can you please clarify the motivation for the change as well ? ( https://cirrus-ci.com/task/5360068607606784 )

File: tautulli.json (hunk 1 above)
Comment (dropsignal) [reply tautulli.json:7]: I've renamed post-install.sh to post_install.sh. https://github.com/arcooke/ returns a 404 error and the plugin install fails. I've recreated the plugin. I do not use NAT so I changed the default to my preference.

**Review by sonicaj — APPROVED** [review]

**Pull request merged**